cmd/brew-command-not-found-init-fish.rb

```diff
@@ -0,0 +1,7 @@
+
+require "fileutils"
+
+file = File.expand_path("#{File.dirname(__FILE__)}/../handler.fish")
+destination = ARGV[0] ? ARGV[0]+"command-not-found.fish" : File.expand_path("~/.config/fish/functions/command-not-found.fish")
```

Comment:
Use `ARGV.empty?` to test if any argument was passed. Also, `ARGV[0]+"command-not-found.fish"` won’t work if `ARGV[0]` doesn’t end with a slash. Use `File.join(ARGV[0], "command-not-found.fish")` to be sure.